packages/cra-template-typescript/template/src/hooks/useFelaEnhanced.ts

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { combineMultiRules } from 'fela-tools';
+import { useFela } from 'react-fela';
+
+// This solution is based on the implementation of [connect binding](https://github.com/robinweser/fela/blob/master/packages/fela-bindings/src/connectFactory.js) from fela
+export default function useFelaEnhanced(rules, { extend, ...otherProps } = {}) {
```

Comment:
It'd be nice if at least function parameters had typings. Ideally from fela package, but I added [custom types on rafflu](https://gitlab.ack.ee/Web/rafflu/-/blob/development/src/hooks/useFelaEnhanced.ts) to set it up fast.

Reply:
Done ✅

Reply:
Great 👍